src/scripts/main.js

```diff
@@ -1 +1,91 @@
 'use strict';
+
+let isClickedLeft = false;
+let isClickedRight = false;
+let timer;
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+function resetTimer() {
+  clearTimeout(timer);
+
+  timer = setTimeout(() => {
+    if (!isClickedLeft) {
+      firstPromise();
+    }
+  }, 3000);
+}
+
+document.addEventListener('mousedown', (e) => {
+  if (e.button === 0) {
+    isClickedLeft = true;
+    firstPromise();
+    resetTimer();
+  }
+
+  if (e.button === 2) {
+    isClickedRight = true;
+  }
+
+  if (isClickedLeft !== isClickedRight) {
+    secondPromise();
+  }
+
+  if (isClickedLeft && isClickedRight) {
```

Comment:
This condition will only be true when both `isClickedLeft` and `isClickedRight` are true. However, it's possible that `thirdPromise` will never be called if the user does not click both mouse buttons at the same time. If this is not the intended behavior, you might want to revise this condition.